src/manifest.rs

```diff
@@ -51,10 +51,13 @@ pub enum OciManifest {
 
 impl OciManifest {
     /// Returns the appropriate content-type for each variant.
-    pub fn content_type(&self) -> &str {
+    pub fn content_type(&self) -> String {
```

Comment:
I may not have enough context - what is the reason the owned object is required here?